mkdocs_swagger_ui_tag/plugin.py

```diff
@@ -56,6 +65,10 @@ class SwaggerUIPlugin(BasePlugin):
         ("validatorUrl", config_options.Type(str, default="none")),
         ("extra_css", config_options.Type(list, default=[])),
         ("dark_scheme_name", config_options.Type(str, default="slate")),
+        (
+            "filter_files",
```

Comment:
I renamed the option to filter_files. I think using the file path is less ambiguous.

Reply:
What about marking files on the markdown frontmatter? (see [here](https://github.com/blueswen/mkdocs-swagger-ui-tag/issues/25#issuecomment-2125575086)) The frontmatter data is already available in the page event hooks. I didn't though about that before, but it feels more ergonomic in general.

Reply:
I'm good with either way